### ISSUE TITLE
NVDA not ignoring images in HTML emails with blank alt text

### ISSUE THREAD

**Issue by tombye** [nvaccess/nvda#9874]
### Steps to reproduce:

1. send a HTML email containing an image with an empty alt attribute
2. open and read the email in the Windows 10 Mail or Outlook 2016 apps

### Actual behavior:

When NVDA gets to the image it announces the URL in its src attribute.

### Expected behavior:

It should interpret the empty alt attribute as an indication that the image is [pure decoration](https://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv-all.html#puredecdef) and ignore it, as in [WCAG 2.0 guideline 1.1.1](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#text-equiv).

#### NVDA installed/portable/running from source:

NVDA is Installed.

#### NVDA version:

2019.1

#### Windows version:

Windows 10, version 1803

#### Name and version of other software in use when reproducing the issue:

Mail app version: 16005.11629.20174.0

#### Other information about your system:

NVDA was running in laptop mode.

### Other questions

Not applicable.

#### Does the issue still occur after restarting your PC?

Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviours.

- NVDA 2018.3.2

**Comment by tombye** [comment]
As an update to this, we have recently tested this on Outlook 2016 with the latest version of NVDA (2020.3) and it now announces "graphic" for images in emails with an empty alt attribute.

This is a great improvement (thank you!) but still doesn't meet WCAG 1.1.1.

**Comment by Adriani90** [comment]
@tombye this is actually ok to report grafic on images without any alt atribute, because then blind people at least notice this could be an image which needs to be described.
Note that many people nowadayse rather use role="presentation" or aria-hidden="true" on purely decorative content rather than empty alt atribute. This is also the recomendation here for example:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/img_role

I am closing this issue as invalid from screen reader perspective.

**Comment by tombye** [comment]
Hello, thanks for your comment @Adriani90 and apologies for posting this so long afterwards. I'm commenting here because I just wanted to be clear about why this issue was closed.

When I used the term 'empty alt attribute', I meant this code `alt=""`. Is that what you understood it to mean too? Your comment seems to refer to images **without** an alt attribute rather than those with one set to an empty string.

I intended this issue to be based on NVDA's behaviour when announcing images with an alt attribute set to an empty string. (My understanding that they should be ignored comes from [the ARIA in HTML spec](https://w3c.github.io/html-aria/#el-img-empty-alt) which says that this gives the image implied `role=presentation` semantics.)
